src/CMakeLists.txt

```diff
@@ -126,20 +126,6 @@ if (WIN32)
     add_definitions(-DNOMINMAX)
 endif()
 
-# determine if VS SDK workload is installed via vswhere.exe 
```

Comment:
now that VS SDK is NuGet-ized, don't have to worry about environment (and don't want to)

Reply:
👍

Reply:
I was unable to build make_cppwinrt_vsix on my devbox. I don't have the VisualStudioExtension workload installed. THis code was checking that so we would bypass building natvis on machines that don't have the correct config. I think that's broken now

Reply:
thanks!  I've migrated the project to using the Microsoft.VSSDK.BuildTools NuGet (although to load it in VS still requires the VS SDK MSI, but builds are at least independent)

Reply:
You mean loading the vsix.sln and/or cppwinrtvisualizer.sln (or assoicated *proj files) in VS requires the VS SDK MSI, right? If I load the cmake project in VS, do I need VS SDK MSI?

Reply:
correct - loading the cmake folder does not load the csproj